RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -237,21 +240,21 @@
 # Propagator taking into account momentum uncertainty in multiple scattering calculation.
 import TrackingTools.MaterialEffects.MaterialPropagatorParabolicMf_cff
 import TrackingTools.MaterialEffects.MaterialPropagator_cfi
-mixedTripletStepPropagator = TrackingTools.MaterialEffects.MaterialPropagator_cfi.MaterialPropagator.clone(
-#mixedTripletStepPropagator = TrackingTools.MaterialEffects.MaterialPropagatorParabolicMf_cff.MaterialPropagatorParabolicMF.clone(
-    ComponentName = 'mixedTripletStepPropagator',
-    ptMin         = 0.1
-)
+mixedTripletStepPropagator = TrackingTools.MaterialEffects.MaterialPropagator_cfi.MaterialPropagator.clone()
+_mixedTripletStepPropagatorParabolicMf = TrackingTools.MaterialEffects.MaterialPropagatorParabolicMf_cff.MaterialPropagatorParabolicMF.clone()
```

Comment:
since we are cloning to a local version, I think that the more appropriate (economic?) variant is to just add one line after the old code
```
trackingParabolicMf.toModify(mixedTripletStepPropagator, SimpleMagneticField = 'ParabolicMf')
```

Reply:
Done in the last update.